angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.component.ts

```diff
@@ -3,397 +3,80 @@ import {
-                          sprk-c-Icon--xl
-                          sprk-u-mrs
-                        "
-                        *ngIf="subNavLink.leadingIcon"
-                      ></sprk-icon>
-                      {{ subNavLink.text }}
-                    </a>
-                  </li>
-                </ul>
-              </sprk-masthead-accordion-item>
-            </div>
-            <div *ngIf="!narrowLink.subNav">
-              <li
-                [ngClass]="{
-                  'sprk-c-MastheadAccordion__item': true,
-                  'sprk-c-MastheadAccordion__item--active': narrowLink.active
-                }"
-              >
-                <a
-                  *ngIf="!narrowLink.routerLink"
-                  sprkLink
-                  variant="unstyled"
-                  class="sprk-c-MastheadAccordion__summary"
-                  [attr.href]="narrowLink.href"
-                  [analyticsString]="narrowLink.analyticsString"
-                >
-                  <span class="sprk-c-MastheadAccordion__heading">
-                    <sprk-icon
-                      [iconType]="narrowLink.leadingIcon"
-                      additionalClasses="
-                        sprk-c-Icon--filled-current-color
-                        sprk-c-Icon--stroke-current-color
-                        sprk-c-Icon--xl
-                        sprk-u-mrs
-                      "
-                      *ngIf="narrowLink.leadingIcon"
-                    ></sprk-icon>
-                    {{ narrowLink.text }}
-                  </span>
-                </a>
-                <a
-                  *ngIf="narrowLink.routerLink"
-                  sprkLink
-                  variant="unstyled"
-                  class="sprk-c-MastheadAccordion__summary"
-                  [routerLink]="narrowLink.routerLink"
-                  [analyticsString]="narrowLink.analyticsString"
-                >
-                  <span class="sprk-c-MastheadAccordion__heading">
-                    <sprk-icon
-                      [iconType]="narrowLink.leadingIcon"
-                      additionalClasses="
-                        sprk-c-Icon--filled-current-color
-                        sprk-c-Icon--stroke-current-color
-                        sprk-c-Icon--xl
-                        sprk-u-mrs
-                      "
-                      *ngIf="narrowLink.leadingIcon"
-                    ></sprk-icon>
-                    {{ narrowLink.text }}
-                  </span>
-                </a>
-              </li>
-            </div>
-          </div>
-        </sprk-masthead-accordion>
-        <ng-content select="[narrowNavFooter]"></ng-content>
-      </nav>
+        <ng-content select="[sprkMastheadBranding]"></ng-content>
+        <ng-content select="[sprkMastheadNavItem]"></ng-content>
+        <ng-content select="[sprkMastheadNavItems]"></ng-content>
+      </sprk-stack>
+      <ng-content></ng-content>
```

Comment:
This blank ng-content is a catch all for the use case where someone would want to use the shell of the masthead but completely write their own markup. They can inject whatever they need simply but NOT using our `sprkMastheadBranding` directive. The assumption is that if they are choosing to not use our branding then they are choosing to go custom.